.goreleaser.yml

```diff
@@ -0,0 +1,26 @@
+before:
+  hooks:
+    - go mod download
+builds:
+  -
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - freebsd
+    goarch:
+      - amd64
+      - arm64
+nfpms:
+  - id: odohd
+    package_name: odohd
+    vendor: Emerald Onion
+    homepage: https://packetframe.com
+    maintainer: Nate Sales <nate@emeraldonion.org>
+    description: Oblivious DNS over HTTPS server
+    license: GNU GPL-3.0
```

Comment:
given the use of MIT code from cloudflare, the easiest is to continue to mark this overall as an MIT package, if that's acceptable to you.

Reply:
Good catch, that's a mistake on my part. Fixed in 9606a688ed75928254f410334b8d32879ecbb721

Reply:
And the hompage there is wrong too. Remnants from an old package. Fixed in 4f4ccc56c894a4339baddbb57c8a564297417ac7